esp32_bridge:
  ros__parameters:
    serial_port: '/dev/ttyUSB0'
    baud_rate: 115200
    killswitch_address: 42
    killswitch_index: 1
    mission_button_address: 43
    mission_button_index: 1
    mission_button_arm_time_ms: 5000